.github/workflows/run_tests_suite_all_latest_frameworks.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    timeout-minutes: 3000
```

Comment:
Is this the tests suit time limit? isn't 30,000 minutes too much?

Reply:
It was just to test it, I set it to 40min.